src/program/downloaders/__init__.py

```diff
@@ -15,11 +15,12 @@ def __init__(self):
         self.key = "downloader"
         self.initialized = False
         self.speed_mode = settings_manager.settings.downloaders.prefer_speed_over_quality
-        self.service = next((service for service in [
-            RealDebridDownloader(),
-            AllDebridDownloader(),
-            #TorBoxDownloader()
-            ] if service.initialized), None)
+        self.services = {
+            RealDebridDownloader: RealDebridDownloader(),
+            AllDebridDownloader: AllDebridDownloader(),
```

Comment:
what does this fix?

Reply:
It fixes the response of the services endpoint. In the endpoint it looks over all the Services (for example Downloader) and looks for children (the `services` property)